client/modules/Slack/SlackBotSender.php

```diff
@@ -41,8 +41,8 @@ class SlackBotSender
      */
     public function __construct($slackBotUrl, $auth = null)
     {
-        $this->slackBotUrl = $slackBotUrl;
-        $this->auth        = $auth;
+        self::$slackBotUrl = $slackBotUrl;
```

Comment:
Ok, great @alebedev80 ! I'm changing it, even though we only have one client for now... Better do it now, while we're on it..